security.md

```diff
@@ -85,6 +86,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2021-3584"></a>CVE-2021-3584: Remote code execution through Sendmail configuration
+
+Sendmail location and arguments, available via Administer - Settings, both accept arbitrary strings and pass them into shell. By default, only Foreman super administrator can access settings.
```

Comment:
```suggestion
Sendmail location and arguments, available via Administer - Settings, both accept arbitrary strings and pass them into shell. By default, only Foreman super administrators can access settings.
```

Reply:
Apologies for not catching this the first time
```suggestion
Sendmail location and arguments, available via Administer - Settings, both accept arbitrary strings and pass them into shell. By default, only Foreman super administrator can access settings. However, non-admin users can be granted the permission to edit settings.
```

Reply:
This is incorrect. By default users with the `Manager` or `System admin` roles also have permission to edit settings.

Reply:
Corrected.